src/pages/Information/Information.jsx

```diff
@@ -1,57 +1,80 @@
 import React from "react";
-import { FormattedMessage } from "react-intl";
+import Typography from "@mui/material/Typography";
+import { useIntl } from "react-intl";
 
-const Information = () => (
-  <>
-    <div data-test="information-display">
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-first-header"
-          defaultMessage="What is HacktoberFest?"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-first-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-      <img
-        data-test="image-display"
-        src="../src/assets/pumpkin_PNG86719.png"
-        alt="hacktoberfest"
-        style={{ height: "100px" }}
-      />
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-second-header"
-          defaultMessage="When & Where"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-second-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-third-header"
-          defaultMessage="Come join the FUN!"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-third-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-  </>
-);
+const Information = () => {
+  const intl = useIntl();
+
+  const infoData = [
+    {
+      header: intl.formatMessage({ id: "Info-first-header" }),
+      defaultMessage: "What is HacktoberFest?",
+    },
+    {
+      paragraph: intl.formatMessage({ id: "Info-first-paragraph" }),
+      defaultMessage: "Lorem ipsum...",
+    },
+    {
+      header: intl.formatMessage({ id: "Info-second-header" }),
+      defaultMessage: "When & Where",
+    },
+    {
+      paragraph: intl.formatMessage({ id: "Info-second-paragraph" }),
+      defaultMessage: "Lorem ipsum...",
+    },
+    {
+      header: intl.formatMessage({ id: "Info-third-header" }),
+      defaultMessage: "Come join the FUN!",
+    },
+    {
+      paragraph: intl.formatMessage({ id: "Info-third-paragraph" }),
+      defaultMessage: "Lorem ipsum...",
+    },
+  ];
```

Comment:
🔧 There should only be one object for each header/paragraph.

Reply:
Addressed:  4fc501bb2d1477550a50115390ccd963961263c4